Pocket/pocket-ios
# fix(layout): adjust iPad layout to use two hero cards

## Summary
Change iPad to use 2 hero cards in layout

## References 
IN-1366

## Implementation Details
* Modify the `heroSection` in `HomeCollectionViewSectionProvider` to account for multiple hero cards when using a wide layout
* Modify data source to populate appropriate items for `recommendationHero` items by creating a property `numberOfHeroItems` on the `HomeViewModel`

## Test Steps
1. Open the app in iPad and iPhone, verify the different layouts
2. Test layout with Split View 

## PR Checklist:
- N / A Added Unit / UI tests

## Screenshots
| iPad | Split View |
| --- | --- | 
| ![Simulator Screenshot - iPad Pro (12 9-inch) (6th generation) - 2023-07-06 at 10 54 57](https://github.com/Pocket/pocket-ios/assets/6743397/7a767284-5049-47ab-a9c4-73c0926e880f)| ![Simulator Screenshot - iPad Pro (12 9-inch) (6th generation) - 2023-07-06 at 10 54 07](https://github.com/Pocket/pocket-ios/assets/6743397/82db19bb-d133-4707-b872-f6d5724c3549) |
